allegro/src/core.rs

```diff
@@ -562,6 +562,17 @@ impl Core
 		unsafe { mem::transmute(al_get_new_bitmap_format() as u32) }
 	}
 
+	 
+	/// This function is used to set the target bitmap that allegro draws to.
+	/// ```
+	/// // How to set target bitmap similar to al_set_target_bitmap
+	/// // For a more complete example look at example/example.rs
+	/// use allegro::*;
+	/// let core = Core::init().unwrap();
+	/// let bmp = Bitmap::new(&core, 400, 400).unwrap();
+	/// core.set_target_bitmap(Some(&map));
```

Comment:
`Some(&bmp)` presumably.

Reply:
That's right. Thanks for catching that. I found a way to test the examples in a rust doc string so when I submit more documentation updates while learning the library I'll run that.